mkosi/sandbox.py

```diff
@@ -98,8 +99,19 @@ def oserror(filename: str = "") -> None:
     raise OSError(ctypes.get_errno(), os.strerror(ctypes.get_errno()), filename or None)
 
 
+UNSHARE_EPERM_MSG = """
+mkosi was forbidden to unshare the user namespace. This probably means your distro has
+restricted unprivileged user namespace creation to reduce exposure to kernel bugs.
+For information about Ubuntu and AppArmor implementations of this restriction,
+see https://ubuntu.com/blog/ubuntu-23-10-restricted-unprivileged-user-namespaces.
+For other systems, research the kernel.unprivileged_userns_clone and
+user.max_user_namespaces sysctls.
+"""
+
 def unshare(flags: int) -> None:
     if libc.unshare(flags) < 0:
+        if ctypes.get_errno() == errno.EPERM:
+            sys.stderr.write(f"\033[0;1;39m{UNSHARE_EPERM_MSG}\033[0m")
```

Comment:
```suggestion
            print(f"\033[0;1;39m{UNSHARE_EPERM_MSG}\033[0m", file=sys.stderr)
```
Also, I'd drop the formatting since I don't think it's really necessary.